dist/kostentraeger-warnings.txt

```diff
@@ -21,7 +21,15 @@ LK06Q321.ke0
   message 18 - skipped invalid VKG 5: Unknown LeistungserbringergruppeSchluessel "3"
   message 18 - skipped invalid VKG 6: Unknown LeistungserbringergruppeSchluessel "3"
 
-AO05Q221.ke1
+EK06Q221.ke3
+  IK 183170002 (hkk) links to IK 183170002 for data but there is not any public key for encryption
+  IK 183170002 (hkk) links to IK 183170002 for data but there is not any public key for encryption
+  IK 183170002 (hkk) links to IK 183170002 for data but there is not any public key for encryption
+  IK 183170002 (hkk) links to IK 183170002 for data but there is not any public key for encryption
+  IK 183170002 (hkk) links to IK 183170002 for data but there is not any public key for encryption
+  IK 183170002 (hkk) links to IK 183170002 for data but there is not any public key for encryption
+
```

Comment:
Right, an additional validity check ("All Datenannahmestellen with decryption capacity must actually have a public key") caused some more mistakes to be uncovered. +sigh+

Will check if this is maybe a bug in the lib and otherwise, ask the ITSG about it, I guess.